pkg/cmd/workspacemode/workspace_mode.go

```diff
@@ -42,7 +50,66 @@ func Execute() {
 	workspaceModeRootCmd.AddCommand(portForwardCmd)
 	workspaceModeRootCmd.AddCommand(exposeCmd)
 
-	if err := workspaceModeRootCmd.Execute(); err != nil {
+	var telemetryService telemetry.TelemetryService
```

Comment:
Can we reuse this block in both `cmd.go` and `workspace_mode.go`? They are almost identical - only thing missing in this one is the interrupt channel but I think it should be included anyway for e.g. `forward` command or other future commands

Reply:
Was trying to figure this out yesterday but couldn't hack it 😅 
Should be good now. Thanks for the bump 😄